packages/components/src/SubHeaderBar/SubHeaderBar.test.js

```diff
@@ -77,7 +76,6 @@ describe('SubHeaderBarActions', () => {
 				<Icon name="talend-bell" />
 			</SubHeaderBarActions>,
 		);
-		expect(wrapper.instance().props.right).toBeTruthy();
```

Comment:
This assertion is useless, it checks that the props we provided is the prop we provided